README.md

```diff
@@ -408,6 +408,7 @@ Simple deployment of [E-mail](https://en.wikipedia.org/wiki/Email) servers, e.g.
 
 - [chasquid](https://blitiri.com.ar/p/chasquid/) - SMTP (email) server with a focus on simplicity, security, and ease of operation. ([Source Code](https://blitiri.com.ar/git/r/chasquid/)) `Apache-2.0` `Go`
 - [Courier MTA](https://www.courier-mta.org/) - Fast, scalable, enterprise mail/groupware server providing ESMTP, IMAP, POP3, webmail, mailing list, basic web-based calendaring and scheduling services. ([Source Code](https://www.courier-mta.org/repo.html)) `GPL-3.0` `C`
+- [EmailRelay](https://emailrelay.sourceforge.net/) - A small and easy to configure SMTP and POP3 server for Windows and Linux. ([Source Code](https://sourceforge.net/projects/emailrelay/)) `GPL-3.0` `C++`
```

Comment:
```suggestion
- [EmailRelay](https://emailrelay.sourceforge.net/) - A small and easy to configure SMTP and POP3 server for Windows and Linux. ([Source Code](https://sourceforge.net/p/emailrelay/git/ci/master/tree/)) `GPL-3.0` `C++`
```

Reply:
Noted, I still approve once this has been addressed as well. 👍🏻

Reply:
currently the repo uses SVN and not fully migrated to git. The the git repo is little bit outdated. I already asked author to migrate to git but he didn't learned it well yet.
I think that just the SF page should be fine.

Reply:
The you should link to the SVN repo instead -> https://sourceforge.net/p/emailrelay/code/HEAD/tree/